package.json

```diff
@@ -242,12 +242,12 @@
         "docker.defaultRegistry": {
           "type": "string",
           "default": "",
-          "description": "Default registry when tagging an image, empty string will target Docker Hub when pushing."
+          "description": "(Deprecated) Default registry when tagging an image, empty string will target Docker Hub when pushing."
```

Comment:
I'd say we should go ahead and remove it so they get errors in the settings file.  You can still query for the value even if it's removed.

Reply:
@fiveisprime your thoughts?

Reply:
I'll go ahead and remove this...